blog/articles/amazon-ses-ec2-rails.md

```diff
@@ -0,0 +1,114 @@
+# Deliver Email with Amazon SES on EC2 for Rails
+
+[Amazon Simple Email Service][ses] (SES) is a
+high availability transactional email service.
+
+[ses]: https://aws.amazon.com/ses/
+
+When sending email from an EC2 instance,
+it is [particularly inexpensive][pricing]:
+
+* $0 for the first 62,000 emails sent per month
+* $0.10 per 1,000 emails sent thereafter
+
+[pricing]: https://aws.amazon.com/ses/pricing/
+
+It is available in three AWS regions:
+
+* eu-west-1 (Ireland)
+* us-east-1 (Northern Virginia)
+* us-west-2 (Oregon)
+
+If you have an AWS account,
+go to [SES in your AWS console][home].
+
+[home]: https://console.aws.amazon.com/ses/home
+
+## Move out of sandbox mode
+
+By default, SES in each region is in sandbox mode.
+Since sandbox mode only allows delivering to
+a whitelist of verified email addresses,
+it's unusable for a production web application.
+
+Go to the "Sending Statistics" page.
+click "Request a Sending Limit Increase"
+to open a support request.
+It may take hours, up to a day or two, for the request to be approved.
+
+[sand]: https://docs.aws.amazon.com/ses/latest/DeveloperGuide/request-production-access.html?icmpid=docs_ses_console
+
+In addition to lifting the restriction on
+delivering only to verified email addresses,
+increasing the sending limit
+will increase the limit from 200 emails/day
+to, for example, 50,000 emails/day.
+
+## Verify an email address
+
+Click "Email Addresses".
+Click "Verify a New Email Address".
+Enter an email such as `support@example.com`
+and click the verification email.
```

Comment:
`and click the verification email.` Is there a button that you can reference here to click? Is it `Verify Email`? Or maybe you meant to say submit the form, then click on the link in your verification email, something like that?